fusesoc/main.py

```diff
@@ -268,7 +280,31 @@ def gen_show(fs, args):
 
 def core_info(fs, args):
     core = _get_core(fs, args.core)
-    print(core.info())
+    print(core.info(fs.config.ssh_trustfile))
+
+
+def core_sign(fs, args):
+    core = _get_core(fs, args.core)
+    logger.info("sign core file: " + core.core_file)
+    logger.info("with key file: " + args.keyfile)
+    sigfile = core.core_file + ".sig"
+    logger.info("put result in: " + sigfile)
+    sig = signature.sign(core.core_file, args.keyfile, None)
+    file = open(sigfile, "w")
+    file.write(sig)
+    file.close()
+    print(f"{sigfile} created")
+
+
+def core_verify(fs, args):
+    core = _get_core(fs, args.core)
+    logger.info("verify core file: " + core.core_file)
```

Comment:
Make these debug messages instead of info

Reply:
Actually, drop them since we already have debug messages saying the same thing in signature.verify(), which gets called by this function